### ISSUE TITLE
Provide description for actor conditions

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#217]
```
This issue is a split from issue 212. Issue created by Samuel.

Samuel stated here:
http://andors.techby2guys.com/viewtopic.php?p=2763#p2763

There is no description about the actor conditions anywhere. IMO all actor 
conditions that are given by items or used by monsters should be transparent. 
Currently I cannot see something like this.
```

Original issue reported on code.google.com by `oskar.wi...@gmail.com` on 15 Jul 2011 at 4:45

**Comment by GoogleCodeExporter** [comment]
```
This is now implemented in r130. 

Thank you for a great suggestion! This was really needed, to give the player 
all info required to make an informed decision about what equipment to use.

Marking this as ready for next release.
```

Original comment by `oskar.wi...@gmail.com` on 15 Jul 2011 at 4:48
- Changed state: **Ready**
- Added labels: **Milestone-0.6.10**

**Comment by GoogleCodeExporter** [comment]
```
Issue 219 has been merged into this issue.
```

Original comment by `oskar.wi...@gmail.com` on 20 Jul 2011 at 11:51

**Comment by GoogleCodeExporter** [comment]
```
Still a bug with negative cc values like BW misery has.
```

Original comment by `SamuelPl...@gmail.com` on 21 Jul 2011 at 8:47

**Comment by GoogleCodeExporter** [comment]
```
re comment 3: Fixed in r140.
```

Original comment by `oskar.wi...@gmail.com` on 23 Jul 2011 at 1:20

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.10 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 21 Oct 2011 at 8:56
- Changed state: **Fixed**
